bin/ch/ch.cpp

```diff
@@ -196,97 +196,42 @@ HANDLE GetFileHandle(LPCWSTR filename)
     return GetStdHandle(STD_OUTPUT_HANDLE);
 }
 
-HRESULT CreateLibraryByteCodeHeader(LPCSTR contentsRaw, JsFinalizeCallback contentsRawFinalizeCallback, DWORD lengthBytes, LPCWSTR bcFullPath, LPCSTR libraryNameNarrow)
+HRESULT CreateLibraryByteCode(LPCSTR contentsRaw)
```

Comment:
With the old process `ch` would generate the entire contents of the file here BUT with my desire to combine multiple JS files into one header file this would have been tricky hence cut this to just generating the bytecode and move the rest of the file creation into python.